src/main/java/seedu/dictionote/model/Model.java

```diff
@@ -76,13 +107,4 @@
      * another existing person in the dictionote book.
      */
     void setPerson(Person target, Person editedPerson);
-
-    /** Returns an unmodifiable view of the filtered person list */
-    ObservableList<Person> getFilteredPersonList();
-
-    /**
-     * Updates the filter of the filtered person list to filter by the given {@code predicate}.
-     * @throws NullPointerException if {@code predicate} is null.
-     */
-    void updateFilteredPersonList(Predicate<Person> predicate);
```

Comment:
may I ask why is this method removed? I notice you added updateFilteredPersonList, but this method was removed totally on the interface.